### DISCUSSION TITLE
Icon request: icon-remote (television)

### DISCUSSION THREAD

**Discussion by napster3000** [FortAwesome/Font-Awesome#21126] [Icon Requests]
Would be great some icons for "Television", "remote control" and so on..

**Comment by tagliala** [comment]
tv is a dup of #368 

renaming as icon-remote

If you have other requst let me know but for indexing and searching purposes it's better a new request for each icon unless they are closely related (e.g.: male/female, thumbs up/thumbs down)

**Comment by ejm** [comment]
+1

**Comment by eevfae** [comment]
+1

**Comment by kevkha** [comment]
+1 As of version 4.4 we do have a television icon (http://fontawesome.io/icon/television/) but there is no remote control. It would be great if you can create a few remote control icons like these https://thenounproject.com/term/remote-control/5491/ - Thanks in advance.

**Comment by derdritte** [comment]
+1

**Comment by aeksco** [comment]
+1

**Comment by sebastien-sebops** [comment]
+1

**Comment by atkawa7** [comment]
+1

**Comment by marcomontresor** [comment]
Would be great to have tv remote control icons - Thanks in advance.

**Comment by MarkSolcom** [comment]
+1

**Comment by AntaresFR** [comment]
I add me to this, it could be nice to have a remote icon.

**Comment by foxymoron** [comment]
+1

**Comment by marcomontresor** [comment]
remote control

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by the-murer** [comment]
+1

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/icons/remote?f=classic&s=solid
